Package.resolved

```diff
@@ -3,19 +3,28 @@
     {
       "identity" : "amazon-location-mobile-auth-sdk-ios",
       "kind" : "remoteSourceControl",
-      "location" : "https://github.com/aws-geospatial/amazon-location-mobile-auth-sdk-ios.git",
+      "location" : "https://github.com/makeen-project/amazon-location-mobile-auth-sdk-ios.git",
```

Comment:
We should not change this - we need to target the official repository.

Reply:
Updated